client_cli/pytests/src/client_cli/have.py

```diff
@@ -70,6 +70,18 @@ def asset_in_assets() -> bool:
         return expected_in_actual(expected, assets)
     return client_cli.wait_for(asset_in_assets)
 
+def asset_quantity(asset_id, expected_quantity):
+    """
+    Check if the expected asset quantity is present in the list of assets.
+
+    :param asset_id: The asset ID.
+    :param expected_quantity: The expected quantity of the asset.
+    :return: True if the asset quantity matches the expected quantity, False otherwise.
+    """
+    return match.iroha_have_asset(
+        expected_quantity,
+        actual=iroha.list_all().assets().get_quantity(asset_id))
```

Comment:
`iroha_have_asset` was removed.

I would suggest to do something like this:

```suggestion
    def assset_has_quantity() -> bool:
        assets=iroha.list_filter(f'{{"Identifiable": {{"Is": "{asset_id}"}}}}').assets()
        actual_quantity = assets.get(asset_id, {})["value"]["Quantity"]
        allure.attach(
            json.dumps(actual_quantity),
            name='actual_quantity',
            attachment_type=allure.attachment_type.JSON)
        allure.attach(
            json.dumps(expected_quantity),
            name='expected_quantity',
            attachment_type=allure.attachment_type.JSON)
        return expected_quantity == actual_quantity
    return client_cli.wait_for(asset_has_quantity)
```